.drone.yml

```diff
@@ -136,6 +136,80 @@ volumes:
 
 ---
 
+kind: pipeline
+type: docker
+name: s390x
+
+platform:
+  os: linux
+  arch: amd64
+
+node: 
+  drone_lab: true
+  arch: s390x
+
+steps:
+  - name: build
+    pull: default
+    image: rohitsakala/dapper:master
```

Comment:
pretty sure these are built now and in the rancher/dapper repo and the manifests are right